### PR TITLE
JSON Logging

### PR DESCRIPTION
**Pull request by mchesler**
Convert all console.log and console.err messages to JSON.  Once deployed, consider setting up fluentd processing of cors proxy logs.

### CODE REVIEW

File: src/cache.coffee [hunk 1]
```diff
@@ -57,7 +57,9 @@ class Cache extends EventEmitter
     Object.keys(@cacheBucket)
 
   log: (message) ->
-    console.log "[CACHE] #{message}" unless @options.logging is false
+    console.log JSON.stringify(
```
Comment (mzuneska) [inline src/cache.coffee]: Maybe we can wrap this up in to a global log function?

**Comment by mnutt** [comment]
Most of the errors I saw in `err` were `ECONNREFUSED` which, while they're JSON, are mulitline which makes it hard to grep. I wonder if we can catch those and display them all on one line.

Other than that, I do see the occasional request aborted stack trace:

```
Error: request aborted
    at IncomingMessage.onAborted (/home/deploy/node_apps/cors-proxy/shared/node_modules/body-parser/node_modules/raw-body/index.js:269:10)
    at emitNone (events.js:67:13)
    at IncomingMessage.emit (events.js:166:7)
    at abortIncoming (_http_server.js:280:11)
    at Socket.serverSocketCloseListener (_http_server.js:293:5)
    at emitOne (events.js:82:20)
    at Socket.emit (events.js:169:7)
    at TCP._onclose (net.js:469:12)
```

**Comment by mzuneska** [comment]
What's the next step that needs to happen with this PR?

**Comment by mnutt** [comment]
I think this is fine to merge as is, we'll just need to do a bit more to get rid of some other non-json errors it throws.

**Pull request merged**